package.json

```diff
@@ -8,7 +8,11 @@
     "start": "next start"
   },
   "dependencies": {
+    "@types/styled-components": "^5.1.9",
```

Comment:
types should be a developer dependency as final build doesn't have types, final build is js files
types are only used in development